packages/local_auth/local_auth_ios/ios/Classes/FLTLocalAuthPlugin.m

```diff
@@ -41,6 +41,10 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     [self deviceSupportsBiometrics:result];
   } else if ([@"isDeviceSupported" isEqualToString:call.method]) {
     result(@YES);
+  } else if ([@"handleAuthReplyWithSuccess" isEqualToString:call.method]) {
```

Comment:
Hm, why was this added?  These should correspond to the messages coming from the dart API, like:
https://github.com/flutter/plugins/blob/2eb616545fff113141f9b0d40851a9322ad54c23/packages/local_auth/local_auth_platform_interface/lib/default_method_channel_platform.dart#L78-L79

Reply:
Oh I added it because  thats how the other tests were testing methods - let me figure out a way to test it without adding that bit then